lua/null-ls/builtins/diagnostics/mypy.lua

```diff
@@ -29,9 +29,10 @@ return h.make_builtin({
         check_exit_code = function(code)
             return code <= 2
         end,
+        multiple_files = true,
         on_output = h.diagnostics.from_pattern(
-            "[^:]+:(%d+):(%d+): (%a+): (.*)  %[([%a-]+)%]", --
-            { "row", "col", "severity", "message", "code" },
+            "([^:])+:(%d+):(%d+): (%a+): (.*)  %[([%a-]+)%]", --
```

Comment:
```suggestion
            "([^:]+):(%d+):(%d+): (%a+): (.*)  %[([%a-]+)%]", --
```

It's strange that this works on your end and not on mine, but this change makes it work for me.

Reply:
Huh. Yea, that should change. Strange that it worked for me... Changed. Maybe I should add a test for mypy.